packages/identity-service/Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM node:18-alpine AS base
```

Comment:
it looks like this is different from the original identity (non-dev) Dockerfile? it's probably worth testing building this and setting the tag on staging

Reply:
Yes this was updated to support the hoisted dependencies in the monorepo setup. It builds and runs locally but I will test on stage!